### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by MarcelGarus**
This PR adds a proper splash screen to the Android app.

Before, the init screens were as follows:

- A white screen (native)
- A white screen with a circular progress indicator (Flutter)
- The actual content (Flutter)

Now, the screens are:

- A screen with an adaptive background color and the logo (native)
- The actual content (Flutter)

That's a better startup experience.

splash screen in dark mode

### CODE REVIEW

File: android/app/src/main/res/drawable/launch_background.xml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Modify this file to customize your launch splash screen -->
-<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:drawable="@android:color/white" />
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android"
+    android:opacity="opaque">
+
+    <item android:drawable="@color/launch_background" />
+    <item android:drawable="@drawable/ic_launcher_foreground" android:gravity="center" />
```
Comment (JonasWanke) [inline android/app/src/main/res/drawable/launch_background.xml]: This doesn't appear to be themed but is always white using the SC flavor.

**Review by JonasWanke — CHANGES_REQUESTED** [review]

**Comment by MarcelGarus** [comment]
I adjusted the splash screen to show the actual app icon in the center, that looks nice in both light and dark mode.
Have a look:

| default theme light | default theme dark | NBC light |
|---|---|---|
| image | image | image |

**Review by JonasWanke — APPROVED** [review]

**Comment by Th3Ph4nt0m** [comment]
Is it on purpose that these changes have not yet been merged?

**Comment by MarcelGarus** [comment]
Nope, thanks for noticing

**Pull request merged**